### DISCUSSION TITLE
Guys what is this problem about and how can i fix this?

### DISCUSSION THREAD

**Discussion by anevkiistefan** [aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE#236] [Q&A] [answered]
Guys when i enter all the stuff i need and when it starts, it gives me captcha and after 5-6 secs it closes my chrome and everything doesn't work. Can someone tell me what i can do to fix that? Here is picture from the problem:

![Capture](https://user-images.githubusercontent.com/78847852/107489913-7c000600-6b89-11eb-9d0e-882431806370.PNG)

**Comment by fakeid30** [comment, marked as answer]
Maybe read the issue tracker before starting a conversation? https://github.com/aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE/issues/233

**Reply by anevkiistefan** [reply]
Oh, i'm new here and didn't saw that, sorry!
